src/content/docs/guides/distro.mdx

```diff
@@ -4,73 +4,84 @@ description: Listings of popular distros for beginners, advanced users etc., as
+        - [elementaryOS](https://elementary.io/) - Distro based on Ubuntu with Pantheon desktop environment
+        - [Linux Lite](https://www.linuxliteos.com/) - Lightweight distribution
+        - [Solus](https://getsol.us/) - Distribution made from scratch, using eopkg package manager
+        - [Puppy Linux](https://puppylinux-woof-ce.github.io/) - Easy to use independent linux distribution
 
- <Tabs>
-    <TabItem label="Debian/Ubuntu-based">
-        - ⭐ **[Debian](https://www.debian.org/)** - Independent Linux distribution
-        - [Damn Small Linux](https://www.damnsmalllinux.org/) - Extremely lightweight Linux distribution
     </TabItem>
-    <TabItem label="Arch-based">
+    <TabItem label="Intermediate">
+        - ⭐ **[Debian](https://www.debian.org/)** - Independent Linux distribution
+        - ⭐ **[openSUSE](https://www.opensuse.org/)** - Easy to learn distribution
         - ⭐ **[EndeavourOS](https://endeavouros.com/)** - Arch-based user-friendly Linux distribution
         - ⭐ **[CachyOS](https://cachyos.org/)** - Fast arch-based distro for gaming and day to day use
+        - [Damn Small Linux](https://www.damnsmalllinux.org/) - Extremely lightweight Linux distribution
         - [ArchCraft](https://archcraft.io/) - Lightweight arch Linux fork with pre-riced window manager
-    </TabItem>
-    <TabItem label="Other">
-        - ⭐ **[openSUSE](https://www.opensuse.org/)** - Easy to learn distribution
         - [Fedora](https://fedoraproject.org/) - Popular desktop distribution
         - [Nobara](https://nobaraproject.org/) - "Gaming" distro based on Fedora
-    </TabItem>
-</Tabs>
 
-## Advanced Distros
-
-<Tabs>
-    <TabItem label="Arch-based">
+    </TabItem>
+    <TabItem label="Advanced">
         - ⭐ **[Arch Linux](https://archlinux.org/)** - Simple and lightweight linux distribution
         - [Artix Linux](https://artixlinux.org/) - No systemd variation of Arch
         - [Parabola](https://www.parabola.nu/) - Distribution approved by Free Software Foundation
-    </TabItem>
-    <TabItem label="Other">
         - ⭐ **[NixOS](https://nixos.org/)** - Distribution based on Nix package manager
         - ⭐ **[Void Linux](https://voidlinux.org/)** - Stable rolling-release distribution using runit
         - [GNU Guix System](https://guix.gnu.org/en/download/) - GNU-developed distribution based on Guix package manager
         - [Linux From Scratch](https://www.linuxfromscratch.org/) - Make your own linux distribution
         - [Gentoo](https://www.gentoo.org/) - Source-based linux distribution
         - [Alpine Linux](https://www.alpinelinux.org/) - Extremely lightweight linux distribution
-        - [Slackware](https://www.slackware.com/)
-        - [bedrock linux](https://bedrocklinux.org/) - Meta linux distribution being able to mix components from others
+        - [Slackware](https://www.slackware.com/) - Independant distro with a pretty old school mentality
```

Comment:
Independant distro with a pretty old school mentality -> **Independent** distro with an **old school approach** (Typo and sentence better written)